pyodk/client.py

```diff
@@ -66,6 +67,11 @@ def __init__(
         self.submissions: SubmissionService = SubmissionService(
             session=self.session, default_project_id=self.project_id
         )
+        self.submission_attachments: SubmissionAttachmentService = (
```

Comment:
Please remove from client - the current use case is to upload submission + attachments at the same time, rather than separately.